packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -99,30 +121,186 @@ export class WorkflowRepository extends Repository<WorkflowEntity> {
 			.execute();
 	}
 
-	async getMany(sharedWorkflowIds: string[], options: ListQuery.Options = {}) {
+	private buildBaseUnionQuery(workflowIds: string[], options: ListQuery.Options = {}) {
+		const subQueryParameters: ListQuery.Options = {
+			select: {
+				createdAt: true,
+				updatedAt: true,
+				id: true,
+				name: true,
+			},
+			filter: options.filter,
+		};
+
+		const columnNames = [...Object.keys(subQueryParameters.select ?? {}), 'resource'];
+
+		const [sortByColumn, sortByDirection] = this.parseSortingParams(
+			options.sortBy ?? 'updatedAt:asc',
+		);
+
+		const foldersQuery = this.folderRepository
+			.getManyQuery(subQueryParameters)
```

Comment:
When are we going to include a filter that not all folders are returned? We should filter by `parentFolderId` so that it is either `NULL` or the given value?

Reply:
Massive overlooked. FIxed it.

Reply:
It was going to be in a next PR but added it here.